src/api/domain/trades-api.ts

```diff
@@ -268,6 +408,20 @@ export const TradesApiFp = function(configuration?: Configuration) {
             const localVarAxiosArgs = await localVarAxiosParamCreator.createTrade(createTradeRequest, xImxEthAddress, xImxEthSignature, authorization, options);
             return createRequestFunction(localVarAxiosArgs, globalAxios, BASE_PATH, configuration);
         },
+        /**
+         * Create a Trade. Currently only buy order is supported.
```

Comment:
> Currently only buy order is supported.

Have we forgotten to update this in the source document?

Reply:
Yeah we need to update this in the source code somehwere in the api. Good catch.

Reply:
We have a ticket in the backlog to audit and update public documentation, we should get on that right after this to iron out any chinks with the docs.